modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/publisher/APIM514CreateAnAPIWithoutProvidingMandatoryFieldsTestCase.java

```diff
@@ -116,12 +118,18 @@ public void testCreateAnAPIThroughThePublisherRestWithoutContext() throws Except
         apiCreationRequestBean.setTags(apiTag);
         apiCreationRequestBean.setDescription(apiDescription);
 
-        HttpResponse apiCreationResponse = apiPublisher.addAPI(apiCreationRequestBean);
-        JSONObject apiResponse = new JSONObject(apiCreationResponse.getData());
-        assertTrue(apiResponse.getBoolean("error"), apiNameTest1 + "can be create without Context");
-        assertTrue(apiResponse.getString("message").contains
-                        ("Context not defined for API"),
-                apiNameTest1 + "can be create without Context");
+        try {
+            APIDTO apiDto = createAndPublishAPI(apiCreationRequestBean, restAPIPublisher, false);
+            assertFalse(StringUtils.isNotEmpty(apiDto.getId()), "Api is not created without proving Context");
+        } catch (ApiException e) {
+            assertEquals(e.getCode(), HttpStatus.SC_INTERNAL_SERVER_ERROR,
+                    "Internal Server Error " );
+        }
+//        JSONObject apiResponse = new JSONObject(apiId);
+//        assertTrue(apiResponse.getBoolean("error"), apiNameTest1 + "can be create without Context");
+//        assertTrue(apiResponse.getString("message").contains
+//                        ("Context not defined for API"),
+//                apiNameTest1 + "can be create without Context");
```

Comment:
Commented code should be removed.(if there's not a special need)

Reply:
3d8ba79